### ISSUE TITLE
Strict mode error only on Node =<0.12 

### ISSUE THREAD

**Issue by phanect** [geddy/geddy#679]
See [this Travis's error](https://travis-ci.org/geddy/geddy/jobs/55230739).

**Comment by phanect** [comment]
Actual error stack trace is following:

```
TypeError: Cannot assign to read only property &#39;controller&#39; of false
    at Object.requestHelpers.getParams (/home/phanect/dev/geddy/lib/app/request_helpers.js:97:25)
    at Domain.&lt;anonymous&gt; (/home/phanect/dev/geddy/lib/app/index.js:307:35)
    at Domain.run (domain.js:197:16)
    at Server.&lt;anonymous&gt; (/home/phanect/dev/geddy/lib/app/index.js:285:13)
    at Server.emit (events.js:110:17)
    at HTTPParser.parserOnIncoming [as onIncoming] (_http_server.js:491:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:23)
    at Socket.socketOnData (_http_server.js:343:22)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
```

Stack trace showing under the test case is just telling that 500 error is occurring, and you can see above error by printing returned HTML.

**Comment by phanect** [comment]
Fixed in #685
